content/applications/productivity/whatsapp.rst

```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+corner. The page will redirect to the :guilabel:`Edit template` page. Here the :guilabel:`Header`,
+:guilabel:`Body`, :guilabel:`Footer` and :guilabel:`Buttons` can be configured. To the right of the
+template is a preview of what the template will look like in production.
+
+.. image:: whatsapp/edit-template.png
+   :align: center
+   :alt: Edit the template using a header, body, footer and buttons.
+
+When all the necessary changes are made to the template, then click on the :guilabel:`Submit` button
+in the upper right hand corner. A confirmation window will appear confirming the Language, click
+:guilabel:`Confirm`. Another confirmation will populate stating that the template will be submitted
+to Meta for review and approval. The :guilabel:`Status` will remain in :guilabel:`In review` until a
+decision has been made by Meta. Once the user receives an email confirming that the template has be
+approved the templates will need to be synced from the Odoo database.
+
+.. _whatsapp/sync:
+
+Syncing templates
+-----------------
+
+Templates must be synced on the Odoo database once they are approved by the Meta team. To do so,
+navigate to :menuselection:`WhatsApp app --> Configuration --> WhatsApp Business Accounts`. Select
+the configuration that should be synced. Under the section marked :menuselection:`Sending messages`,
+towards the bottom, click on :guilabel:`Sync Templates`. Meta will update the templates that are
+approved so that they can be utilized with various apps in the database.
+
+.. image:: whatsapp/sync-template.png
+   :align: center
+   :alt: Syncing Meta WhatsApp templates to the Odoo database, with the 'Sync Templates'
+         highlighted.
+
+A successful message in green should populate in the upper right hand corner with the number of
+templates updated.
+
+Using variables
+---------------
+
+Dynamic variables can be input on the WhatsApp template on the Meta business manager. This process
+should be completed during the initial layout and design through the Meta business console. Dynamic
+variables reference certain fields within the Odoo database to produce unique data in the WhatsApp
+message through the template. Dynamic variables are encoded to display fields from within the
+database referencing fields from within a model.
+
+.. example::
+   Many companies like to customize their WhatsApp messages with a personalized piece of customer
+   information to grab attention. This can be accomplished in Odoo by referencing a field within a
+   model by settings a dynamic variable. For example, a customer's name can be referenced in the
+   email from the :guilabel:`Customer` -->  field on the :guilabel:`Sales Order` model.
+
+First navigate to  `<https://business.facebook.com/wa/manage/home>`_. Then go to
+:menuselection:`Account tools --> Message templates`. Click on one and then :guilabel:`Edit
+template` or create a new template.
```

Comment:
```suggestion
template`.  If a new template is needed, create a new template by clicking :guilabel: `Create`.
```

I can't follow along here, so I am not sure if this is correct- but if the default/usual way is to edit a template maybe stick to just that scenario, or add a note hat this can be done on a new template?

Reply:
Should be 'new'